sqalx_test.go

```diff
@@ -197,7 +198,7 @@ func testSqalxNestedTransactions(t *testing.T, testSavePoint bool) {
 	err = n1_1.Commit()
 	require.Equal(t, sqalx.ErrNotInTransaction, err)
 
-	n1_1, err = n1.Beginx()
+	n1_1, err = n1.BeginTxx(context.Background(), nil)
```

Comment:
Shouldn't we test both `Beginx` and `BeginTxx`?

Reply:
It's done on line 188 :)